assets/src/edit-story/elements/image/display.js

```diff
@@ -26,24 +25,17 @@ const Element = styled.div`
 `;
 
 const Img = styled.img`
-	position: relative;
+	position: absolute;
 	${ ImageWithScale }
 `;
 
-function ImageDisplay( { id, src, origRatio, width, height, x, y, scale, offsetX, offsetY, rotationAngle, forwardedRef, onPointerDown } ) {
-	// Width and height are percent of the actual page dimensions,
-	// Thus 20-by-20 doesn't mean square, but "same as page ratio".
-	const actualRatio = width / height * PAGE_WIDTH / PAGE_HEIGHT;
-	const imgProps = getImgProps( scale, offsetX, offsetY, origRatio, actualRatio );
+function ImageDisplay( { id, src, origRatio, width, height, x, y, scale, focalX, focalY, rotationAngle, isFullbleed, forwardedRef, onPointerDown } ) {
 	const elementProps = {
-		width,
-		height,
-		x,
-		y,
-		rotationAngle,
+		...getBox( { x, y, width, height, rotationAngle, isFullbleed } ),
```

Comment:
Currently, after resizing an Image, the full-bleed doesn't work as expected anymore. Although the attributes for the `div` (`Element`) are updated, the actual style is not updated.
To reproduce: resize an image and then try to set as full-bleed.

Reply:
Great catch! The issue is the conflict of styles between `Moveable` and `SetProperties`. `Moveable` works via inline styles and `SetProperties` via stylesheet (via styled elements). Thus the `Moveable` with inline styles always takes precedence. Fixed by resetting inline styles after `Moveable` is done.